util/auth.go

```diff
@@ -24,11 +23,12 @@ type Authenticator interface {
 }
 
 type auth struct {
-	key string
+	Skey string 	// Signin Key
+	Akey string		// Auth Key
```

Comment:
Does `Skey` and `Akey` used in other packages ?  Other than `util` ?

Reply:
hmm no. I should have made them private